google/cloud/pubsub/internal/streaming_subscription_batch_source.cc

```diff
@@ -236,16 +236,22 @@ void StreamingSubscriptionBatchSource::ExtendLeases(
   }
   lk.unlock();
   for (auto& r : split) {
+    lk.lock();
+    auto modack_request_id = modack_request_id_++;
```

Comment:
It feels like the member variable would be better named as `next_modack_request_id_`.

Reply:
Done.

Reply:
Changed. Well no longer needed.